src/styles/index.scss

```
@@ -550,7 +551,8 @@ input[type='email']:focus {
 		max-width: 700px;
 	}
 	p {
-		@include para(1.1rem, 1.3rem);
+		@include para(1.1srem, 1.8rem);
```

Let's keep these consistent between components.
```suggestion
		@include para(1rem, 1.6rem);
```